packages/devextreme/js/ui/chat.d.ts

```diff
@@ -50,7 +50,7 @@ export type User = {
      * @default undefined
```

Comment:
Seems, that we shouldn't say that default value is undefined

Reply:
According to the API Reviwers solution, the default type will be written as string